packages/freezed/lib/src/tools/type.dart

```diff
@@ -73,6 +73,10 @@ String resolveFullTypeStringFrom(
     }
   }
 
+  if (displayType.contains('InvalidType')) {
+    displayType = displayType.replaceAll('InvalidType', 'dynamic');
+  }
```

Comment:
There must be a better way to do this. Rather than matching for the `InvalidType` string, it would be better to check that the type is an `InvalidType` dart type